{
	"header": {
		"line": "CVE-2019-15522 (An issue was discovered in LINBIT csync2 through 2.0. csync_daemon_ses ...)",
		"name": "CVE-2019-15522",
		"description": "(An issue was discovered in LINBIT csync2 through 2.0. csync_daemon_ses ...)"
	},
	"annotations": [
		{
			"line": "\t- csync2 2.0-25-gc0faaf9-1 (bug #955445)",
			"type": "package",
			"package": "csync2",
			"kind": "fixed",
			"version": "2.0-25-gc0faaf9-1",
			"flags": [
				{
					"bug": 955445
				}
			]
		},
		{
			"line": "\t[buster] - csync2 2.0-22-gce67c55-1+deb10u1",
			"type": "package",
			"release": "buster",
			"package": "csync2",
			"kind": "fixed",
			"version": "2.0-22-gce67c55-1+deb10u1"
		},
		{
			"line": "\t[stretch] - csync2 2.0-8-g175a01c-4+deb9u1",
			"type": "package",
			"release": "stretch",
			"package": "csync2",
			"kind": "fixed",
			"version": "2.0-8-g175a01c-4+deb9u1"
		},
		{
			"line": "\t[jessie] - csync2 <no-dsa> (Minor issue)",
			"type": "package",
			"release": "jessie",
			"package": "csync2",
			"kind": "no-dsa",
			"description": "Minor issue"
		},
		{
			"line": "\tNOTE: https://github.com/LINBIT/csync2/pull/13/commits/0ecfc333da51575f188dd7cf6ac4974d13a800b1",
			"type": "NOTE",
			"description": "https://github.com/LINBIT/csync2/pull/13/commits/0ecfc333da51575f188dd7cf6ac4974d13a800b1"
		}
	]
}
